app/scripts/modules/core/src/search/searchResult/Renderers.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { startCase, kebabCase } from 'lodash';
+
+import { NgReact } from 'core/reactShims';
+
+export interface ISearchColumn {
+  key: string;
+  label?: string;
+  notSortable?: boolean;
+  scopeField?: boolean;
+}
+
+/****** Layout Renderers ******/
+
+export class TableHeader extends React.Component {
```

Comment:
These Renderer components are used by individual `ISearchResultType`s and will help keep the class names consisten